libr/core/disasm.c

```diff
@@ -2172,6 +2172,31 @@ static int flagCmp(const void *a, const void *b) {
 	return strcmp (fa->name, fb->name);
 }
 
+static void __preline_flag(RDisasmState *ds, RFlagItem *flag) {
+	ds_newline (ds);
+	ds_begin_line (ds);
+	ds_pre_line (ds);
```

Comment:
One day there should proper documentation as to how to begin a line and how to print comments, but right now the tests will have to do.